src/test/java/com/googleinterns/zoomtube/servlets/IconFeedbackServletTest.java

```diff
@@ -112,4 +129,61 @@ public void doPost_validRequest_shouldStoreInDatabase() throws IOException, Serv
     assertThat(iconFeedback.timestampMs()).isEqualTo(456);
     assertThat(iconFeedback.type().toString()).isEqualTo("GOOD");
   }
+
+  @Test
+  public void doGet_missingLectureId_shouldRespondWithBadRequest()
+      throws IOException, ServletException {
```

Comment:
Fola mentioned on another one of my PRs that we should all be doing `throws Exception` for tests

Reply:
Lets do this all at one in a future pr

Reply:
sounds good